hbp_knowledge/er-stress/doyle2011.bel

```diff
@@ -271,36 +306,40 @@ caspases at the onset of apoptosis. Specifically it processes procaspase-
 9 to its active form, which then activates downstream
 effector caspases including caspase-3, -7 and -6 [24], leading to
 apoptosis."
-
+#doesnt downstream activation mean that CASP9 -> 3 -> 7 -> 6?
+SET Confidence = "Low"
```

Comment:
No. I had the same doubt and I checked the reference paper. It suggested that  casp9 individually activates others casp.